Chinese/YingJeouMa.md

```diff
@@ -0,0 +1 @@
+成為某國總統，禍國殃民。
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.